stracker/items/enums.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ItemTypes(models.TextChoices):
```

Comment:
Type lar enum mu olmalı?

Reply:
Yani belli bir standartta olsun istedim. Gerçi bunun için ayrı bir model de olabilir. Bu kısmı tartışabiliriz.